src/components/VmDetails/cards/UtilizationCard/UtilizationCharts/DonutChart.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { ChartDonut } from '@patternfly/react-charts'
+
+const DonutChart = ({ data, title, subTitle, id }) => {
+  return (
+    <div id={id}>
```

Comment:
````jsx

const DonutChart = ({ data, title, subTitle, id }) => {
  return (
    <div id={id} style={{ width: '160px', height: '160px', alignSelf: 'center' }}>
      <ChartDonut
        width={160}
        height={160}
        colorScale={['rgb(0, 136, 206)', 'rgb(209, 209, 209)']}
        data={data}
        labels={datum => datum.label}
        subTitle={subTitle}
        title={title}
      />
    </div>
  )
}
````

That makes the donuts look less compressed and the labels are readable:
![screenshot-localhost-3000-2019 07 03-17-04-43](https://user-images.githubusercontent.com/3985964/60625015-a9cd7800-9db4-11e9-93ce-a0368920d9c3.png)

Reply:
Yes, but we cannot hardcode size of charts, because it wouldn't be responsive. But ok, I'll try to play more with sizes.

Reply:
I wouldn't expect the size of the charts to change as the page gets bigger or smaller.  The relative layouts sure.  Changing the chart sizes would be similar to changing the font sizes as the screen is compressed. I don't think we do that currently...